src/workflow/workflow.js

```diff
@@ -269,6 +269,11 @@ const pinAndSetImageCid = async (wfConfig) => {
 
 const setInstanceMetadata = async (wfConfig) => {
   // 6- set metadata for instances
+  const instanceMetadata = wfConfig.instance.metadata;
+  if (!instanceMetadata && typeof instanceMetadata !== 'object' && !Object.keys(instanceMetadata).length) {
+    return;
```

Comment:
Might be usefull to print some output to let the user know this step is not configured and is skipped. e.x. 'Skipped! No instance metadata is configured for the workflow'. Probably all outputs will be revampped/organized so it can also be handled as part of #3.

Reply:
Good point!